core/src/main/java/greencity/exception/handler/CustomExceptionHandler.java

```diff
@@ -158,13 +159,13 @@ public final ResponseEntity<Object> handleRemoteServerUnavailableException(WebRe
     }
 
     /**
-     * Exception handler for {@link MappingException}.
+     * Exception handler for {@link MappingException} {@link CourierAlreadyExists}.
      *
      * @param request {@link WebRequest} with error details.
      * @return {@link ResponseEntity} with http status and exception message.
      */
-    @ExceptionHandler(MappingException.class)
-    public final ResponseEntity<Object> handleModelMapperMappingException(WebRequest request) {
+    @ExceptionHandler({MappingException.class, CourierAlreadyExists.class})
+    public final ResponseEntity<Object> handleModelMapperMappingAndCourierAlreadyExistsException(WebRequest request) {
```

Comment:
please create a new handler for `CourierAlreadyExists` exception
And please rename `CourierAlreadyExists` -> `CourierAlreadyExistsException`